### PR TITLE
Development

### PR DESCRIPTION
**Pull request by ioilmio**
This is a weather web app is consuming OpenWeatherMap API to get weather city's info and Pexels API for the background image;

It is built using:

Javascript ES6
Webpack
Scss
EsLint
VsCode
Please check the previous PR for more info

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hello again @ioilmio 👋

**Good work here. 💪**

- [ ] However, you have your merge request all wrong. You should be seeking to merge a feature branch into development, not the other way around. Kindly fix this.

- [ ] Please check the comment on your code.

### **STATUS: CHANGES REQUESTED** 👷‍♂️

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

HAPPY CODING👨‍💻

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    "sourceType": "module"
+  },
+  "extends": ["airbnb-base"],
+  "rules": {
+    "no-shadow": "off",
+    "no-param-reassign": "off",
+    "eol-last": "off",
+    "arrow-parens": "off",
+    "import/extensions": "off",
+    "import/no-unresolved": "off"
```
Comment (mikenath223) [inline .eslintrc.json:19]: - [ ] You shouldn't add new rules here. Rather disable this rule in each javascript file in the actual affected line. Please remove these added lines here.

**Comment by ioilmio** [comment]
Reviewer, please check the discussion with previous TSEs about GitFlow, I also talked with Full-time TSE and he knows about that.

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hello @ioilmio 👋👋

**Nice job fixing some issues raised in the previous review.**

- [ ] However, as stated in the previous review you have your PR all wrong. You shouldn't have worked in the development branch directly. Rather you should have worked in a feature branch and then open a PR to merge the feature branch in the development branch. But you did this the other way round. To fix this perhaps **rename the weather branch to development, next rename the development branch to weather**.

### **STATUS: CHANGES REQUESTED** 👷‍♂️

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

HAPPY CODING👨‍💻

**Pull request closed without merging**